.rubocop.yml

```diff
@@ -11,7 +11,7 @@ LineLength:
 # Increase allowed lines in a method.  Short methods are good, but 10 lines
 #   is a bit too low.
 MethodLength:
-  Max: 40
+  Max: 60
```

Comment:
What's the reason this was increased? If there's a good reason, this should probably be changed in our templates too.